litigation_data_mapper/parsers/family.py

```diff
@@ -464,14 +470,58 @@ def map_families(
     return mapped_families
 
 
+def add_root_us_principal_law_concept(
+    family_concepts: list[dict[str, Any]], concepts: dict[int, Concept]
+):
+    """
+    Adds the synthetic US principal law concept to the family concepts list
+    if any principal law relationship is present in the family.
+
+    :param list[dict[str, Any]] family_concepts: A list of concept dictionaries that form a family group.
+    :param dict[int, Concept] concepts: A dictionary mapping concept IDs to Concept objects.
+
+    :return list[dict[str, Any]]: The updated list of family concepts.
+    """
+
+    us_principal_law = concepts.get(US_ROOT_PRINCIPAL_LAW_ID)
+
+    if us_principal_law is None:
+        click.echo(
+            f"🛑 US principal law concept with ID {US_ROOT_PRINCIPAL_LAW_ID} not found in concepts."
+        )
+        return family_concepts
```

Comment:
When do we get into this state? And should we error as we really need this to be valid data?